src/sources/nats.rs

```diff
@@ -45,13 +45,20 @@ pub struct NatsSourceConfig {
     /// The NATS URL to connect to.
     ///
     /// The URL must take the form of `nats://server:port`.
+    #[configurable(metadata(docs::examples = "nats://demo.nats.io"))]
+    #[configurable(metadata(docs::examples = "nats://127.0.0.1:4222"))]
     url: String,
 
     /// A name assigned to the NATS connection.
     #[serde(alias = "name")]
     connection_name: String,
 
     /// The NATS subject to pull messages from.
+    #[configurable(metadata(docs::examples = "foo"))]
+    #[configurable(metadata(docs::examples = "time.us.east"))]
+    #[configurable(metadata(docs::examples = "time.*.east"))]
+    #[configurable(metadata(docs::examples = "time.>"))]
+    #[configurable(metadata(docs::examples = ">"))]
```

Comment:
🤔 what does `>` signify?

Reply:
From [here](https://docs.nats.io/nats-concepts/subjects#matching-multiple-tokens) the wildcard `*` matches a single token and `>` matches multiple tokens. So I'm guessing `>` matches everything.

Reply:
I'll put that link into the docs.

Reply:
Yeah, I guess it falls into the realm of someone using NATS would recognize it (probably) - adding a link seems like a good addition.